util/venv-dependencies.sh

```diff
@@ -36,6 +36,7 @@ requires sudo/root access.
 'new-directory' must be somewhere you can write to. A good place may be
 $HOME/streisand-deps. If it already exists,
 please delete the directory, or use a different name.
+
```

Comment:
Intentional whitespace diff?

Reply:
Yeah. The style for all the other diagnostics has a trailing blank line.